api_app/db/migrations/resources.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import uuid
+from azure.cosmos import CosmosClient
+from db.errors import EntityDoesNotExist
+from db.repositories.operations import OperationRepository
+from db.repositories.resources import ResourceRepository
+
+
+class ResourceMigration(ResourceRepository):
+    def __init__(self, client: CosmosClient):
+        super().__init__(client)
+
+    def add_deployment_status_field(self, operations_repository: OperationRepository) -> bool:
+
+        for op in operations_repository.query("SELECT * from c ORDER BY c._ts ASC"):
+            try:
+                resource = self.get_resource_by_id(uuid.UUID(op['resourceId']))
+                resource.deploymentStatus = op['status']
+                self.update_item(resource)
+            except EntityDoesNotExist:
+                logging.info(f'Resource Id {op["resourceId"]} not found')
+                # ignore errors and try the next one
+
+        return True
```

Comment:
nit: Is it worth returning a boolean if it's always going to be true? You never test for it. In the calling code you don't test and always set a status of Executed? Is it worth returning a number of how many rows have been updated so you could add that into the status. That might be useful